### ISSUE TITLE
How to output video in real time?

### ISSUE THREAD

**Issue by HyeonMooJeon** [AlexeyAB/darknet#2945]
hi I'm Students studying about this
I want to save video in real time to webcam and ipCarmera Where can I find it? 
I want only using the code
I used the API, but I do not want it  because the API is very slow

**Comment by rveck** [comment]
You might use OpenCV library

**Comment by AlexeyAB** [comment]
@HyeonMooJeon Hi

Web-cam:
`./darknet detector demo data/coco.data yolov3.cfg yolov3.weights -c 0 -out_filename result.avi`

IP-cam:
`./darknet detector demo data/coco.data yolov3.cfg yolov3.weights http://192.168.0.80:8080/video?dummy=param.mjpg -out_filename result.avi`

Video-file:
`./darknet detector demo data/coco.data yolov3.cfg yolov3.weights test.mp4 -out_filename result.avi`


More: https://github.com/AlexeyAB/darknet#how-to-use-on-the-command-line

**Comment by dahlan77** [comment]
can we push (the result's detection) real time in website?

**Comment by 2017TJM** [comment]
The final  result is bad ,so I want't to do that.  




------------------ 原始邮件 ------------------
发件人: "dahlan77"<notifications@github.com>;
发送时间: 2019年5月6日(星期一) 上午9:31
收件人: "AlexeyAB/darknet"<darknet@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>; 
主题: Re: [AlexeyAB/darknet] How to output video in real time? (#2945)




can we push (the result's detection) real time in website?

**Comment by AlexeyAB** [comment]
@dahlan77 Yes, you can see video with detections on the Web-site: https://github.com/AlexeyAB/darknet/issues/2969#issuecomment-484971112

**Comment by HyeonMooJeon** [comment]
thank you AlexeyAB

**Comment by dahlan77** [comment]
I mean that not only publish into web but aggregate data from json into pie chart or graph,any idea @AlexeyAB ,is it true by using crow ?

Thank you for your helpingg
